src/io/OutputWriter.cpp

```diff
@@ -343,6 +344,14 @@ odc::io::ReceiverWriter::ReceiverWriter( char *inputFileName, char *outputFileNa
   }
   m_numberOfReceivers = atoi( line );
 
+  //! Check for TAINTED_SCALAR (Coverity Scan)
```

Comment:
Do not mention Coverity Scan in source code:
 1) If the issue is real, we fix it.
 2) If not, we suppress it.